internal/controller/suite_test.go

```diff
@@ -164,3 +198,23 @@ func downloadCRD(url string, downloadDir string) {
 	_, err = io.Copy(out, resp.Body)
 	Expect(err).ToNot(HaveOccurred())
 }
+
+func getNumaflowControllerDefinitions() config.NumaflowControllerDefinitionConfig {
+	// Read definitions config file
+	// TODO: use this file instead "../../tests/config/controller-definitions-config.yaml"
```

Comment:
why not use this file in this PR then?

Reply:
Will leave this for @chandankumar4 PR to avoid conflicts.